docs/posix-secondaries.adoc

```diff
@@ -0,0 +1,86 @@
+= Posix Secondaries aka IP Secondaries
+
+Secondary is a term introduced by link:https://uptane.github.io/uptane-standard/uptane-standard.html[the uptane spec].
```

Comment:
Uptane isn't the origin of the term "secondary", but I think referencing the spec is not a bad idea. I'd suggest moving this first paragraph after the second ("The goal of this doc...") and then saying that the term secondary is being used as it is defined by the Uptane standard.

Reply:
Yeah, the origin of the term is lost in the mists of time... :). 
NP, I'll update the doc as you suggested.